resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -7,21 +7,29 @@ restartfile="/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/resta
 start_timeout="${START_TIMEOUT:-<%= EZBake::Config[:start_timeout] %>}"
 
 realname="<%= EZBake::Config[:real_name] %>"
-PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
+rundir="/var/run/puppetlabs/${realname}"
+PIDFILE="${rundir}/${realname}.pid"
 
 if [ ! -e "${INSTALL_DIR}/ezbake-functions.sh" ]; then
     echo "Unable to find ${INSTALL_DIR}/ezbake-functions.sh script, failing start." 1>&2
     exit 1
 fi
 
+/usr/bin/install --directory --owner=$USER --group=$GROUP --mode=755 "$rundir"
```

Comment:
We inherit them from the sourcing the /etc/sysconfig|default in the upstream app command.  Maybe it would be better to fall back to the default from the Ezbake config in the event they weren't there, though.